<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSettingPanel</class>
 <widget class="QWidget" name="SystemSettingPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QPushButton" name="pb_update">
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>237</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>G00/点动参数</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>抬刀固定参数</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>最大加速度(um/s^2)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sb_pb_maxacc">
        <property name="maximum">
         <number>9000000</number>
        </property>
        <property name="singleStep">
         <number>50000</number>
        </property>
        <property name="value">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>最大加速度(um/s^2)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="sb_jump_maxacc">
        <property name="maximum">
         <number>9000000</number>
        </property>
        <property name="singleStep">
         <number>50000</number>
        </property>
        <property name="value">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>NACC(ms)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sb_pb_nacc">
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>NACC(ms)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="sb_jump_nacc">
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>MFR0(um/s)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="sb_pb_mfr0">
        <property name="maximum">
         <number>90000</number>
        </property>
        <property name="singleStep">
         <number>1000</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>抬刀缓冲(um)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="sb_jump_buffer">
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>MFR1(um/s)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="sb_pb_mfr1">
        <property name="maximum">
         <number>90000</number>
        </property>
        <property name="singleStep">
         <number>1000</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>MFR2(um/s)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="sb_pb_mfr2">
        <property name="maximum">
         <number>90000</number>
        </property>
        <property name="singleStep">
         <number>1000</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>MFR3(um/s)</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="sb_pb_mfr3">
        <property name="maximum">
         <number>90000</number>
        </property>
        <property name="singleStep">
         <number>1000</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>237</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pb_save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
